package.json

```diff
@@ -29,6 +31,7 @@
   "homepage": "https://github.com/pelias/dbclient",
   "devDependencies": {
     "intercept-stdout": "^0.1.2",
+    "npm-check": "git://github.com/orangejulius/npm-check.git#disable-update-check",
```

Comment:
Surely it is possible, but npm install and pre-commit tests seem to work fine here.